src/UIKit/UITextField.cs

```diff
@@ -45,6 +45,14 @@ public UITextFieldEditingEndedEventArgs (UITextFieldDidEndEditingReason reason)
 	///     <related type="sample" href="https://github.com/xamarin/ios-samples/tree/master/MonoCatalog-MonoDevelop/">monocatalog</related>
 	public delegate bool UITextFieldCondition (UITextField textField);
 
+	/// <param name="textField">To be added.</param>
+	///     <param name="ranges">To be added.</param>
+	///     <param name="replacementString">To be added.</param>
+	///     <summary>A delegate used to respond to changes on the UITextField.</summary>
+	///     <returns>To be added.</returns>
+	///     <remarks>To be added.</remarks>
```

Comment:
This can be tidied up a bit:


```suggestion
	/// <summary>A delegate used to respond to changes on the UITextField.</summary>
	/// <param name="textField">To be added.</param>
	/// <param name="ranges">To be added.</param>
	/// <param name="replacementString">To be added.</param>
```

Also provide actual documentation instead of "To be added." (or remove it).